braccio3_urdf:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers --------------------------------------
  arm_controller:
    type: velocity_controllers/JointTrajectoryController
    joints:      
      - Giunto0
      - Giunto1
      - Giunto2
      - Giunto3
      - Giunto4
      - Giunto5


    gains:
      Giunto0: {p: 2.0287, d: 0.1009, i: 10.0943,}
      Giunto1: {p: 2.0287, d: 0.1009, i: 10.0943,}
      Giunto2: {p: 2.0287, d: 0.1009, i: 10.0943,}
      Giunto3: {p: 2.0287, d: 0.1009, i: 10.0943,}
      Giunto4: {p: 2.0287, d: 0.1009, i: 10.0943,}
      Giunto5: {p: 2.0287, d: 0.1009, i: 10.0943,}